db/migrate/20240109152921_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: { to_table: :users }
+      t.string :title
+      t.text :text
+
+      t.timestamps
+    end
```

Comment:
- [ ] Kindly add the `likes_counter` and `posts_counter` columns to the posts table.